Servers/UI/OJS.Servers.Ui/ClientApp/src/hooks/submissions/use-submissions-details.tsx

```diff
@@ -137,46 +117,22 @@ const SubmissionsDetailsProvider = ({ children }: ISubmissionsDetailsProviderPro
         [ getProblemResultsRequest, startLoading, stopLoading, submissionResultsByProblemUrlParams ],
     );
 
-    useEffect(() => {
-        if (isNil(apiProblemResults)) {
-            return;
-        }
-
-        setCurrentProblemSubmissionResults(apiProblemResults);
-    }, [ apiProblemResults ]);
-
-    const {
-        get: getProblemResultsByUserRequest,
-        data: apiProblemResultsByUser,
-    } = useHttp({
-        url: getSubmissionResultsByProblemAndUserUrl,
-        parameters: submissionResultsByProblemAndUserUrlParams,
-    });
-
     useEffect(
         () => {
-            if (isNil(submissionResultsByProblemAndUserUrlParams)) {
+            if (isNil(apiProblemResults)) {
                 return;
             }
 
-            (async () => {
-                startLoading();
-                await getProblemResultsByUserRequest();
-                setSubmissionResultsByProblemAndUserUrlParams(null);
-                stopLoading();
-            })();
+            if (!isNil(apiProblemResultsError)) {
+                setProblemResultsError(apiProblemResults as unknown as IErrorDataType);
```

Comment:
is this needed here 'as unknown as' IErrorDataType?

Reply:
I need to cast it to unknown first to be able to cast it to IErrorDataType.